Memorize/CardView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CardView.swift
+//  Memorize
+//
+//  Created by zhira on 7/9/24.
+//
+
+import Foundation
+import SwiftUI
+
+struct CardView : View {
+    @State var isFaceUp: Bool = false
+    let content: String
+    
+    var body: some View {
+        ZStack {
+            let base = RoundedRectangle(cornerRadius: 12)
+            Group {
+                base.fill(.white)
+                base.strokeBorder(lineWidth: 2.0)
+                Text(content).font(.largeTitle)
+            }.opacity(isFaceUp ? 1 : 0)
+            base.fill().opacity(isFaceUp ? 0.0 : 1.0)
+        }
+        .onTapGesture {
+            isFaceUp.toggle()
+        }
+    }
+}
```

Comment:
Most of the comments that I have are just style comments. Normally this is actually not bad, but to improve readability would be better to just separate, modifiers and view more clearly like:

```swift
struct CardView : View {
    @State var isFaceUp: Bool = false
    let content: String

    var body: some View {
        ZStack {
            let base = RoundedRectangle(cornerRadius: 12)
            Group {
                base
                   .fill(.white)
                   
                base
                  .strokeBorder(lineWidth: 2.0)
                  
                Text(content)
                    .font(.largeTitle)
            }
            .opacity(isFaceUp ? 1 : 0)
            
            base
                .fill()
                .opacity(isFaceUp ? 0.0 : 1.0)
        }
        .onTapGesture {
            isFaceUp.toggle()
        }
    }
}

Reply:
Thanks I will fix the changes in the next PR that includes the second assignment.